client/extensions/woocommerce/state/ui/products/api-plan/actions.js

```diff
@@ -9,26 +13,32 @@ import {
 	WOOCOMMERCE_EDIT_PRODUCT_APIPLAN_STEP_END,
 } from '../../../action-types';
 
-/* TODO: Create the plan based on our current edits.
-export function createApiPlanForEdits( productCategoryEdit, productEdits, variationEdits ) {
-	const plan = [];
+// TODO: Eventually think about edits that could belong to multiple sites
+// (e.g. moving a product from one site to another)
+export function createApiPlanForEdits( siteId, productEdits /*, variationEdits = null, productCategoryEdits = null*/ ) {
```

Comment:
I think the intention is that a subsequent PR will add variation plan step and category plan step generation in this action?